README.md

```diff
@@ -70,6 +70,10 @@ The second set of digits is the encryption key. The Excalidraw server doesn’t
 
 Find a growing list of libraries containing assets for your drawings at [libraries.excalidraw.com](https://libraries.excalidraw.com).
 
+## Embed Excalidraw in your App?
```

Comment:
```suggestion
## Building an app using Excalidraw?
```

If not the above, let's change `Embed` → `Embedding`?

Reply:
The latter sounds better :)